src/metabase/analytics/stats.clj

```diff
@@ -124,26 +124,44 @@
     (config/config-str :database-url)        :heroku ;; Putting this last as 'database-url' seems least specific
     :else                                    :unknown))
 
+(defn appearance-ui-colors-changed?
+  "Returns true if the 'User Interface Colors' have been customized"
+  []
+  (boolean (not-empty (select-keys (public-settings/application-colors) [:brand :filter :summarize]))))
```

Comment:
Thanks, I ended up with seq as with `(boolean (not-empty` I got
`use the idiom (seq x) rather than (not (empty? x))clj-kondo(not-empty?)`

Reply:
Right, `seq` is the idiomatic way to test if a sequence is not empty. In this case of course that's not what we want because we really need a boolean in the end. But you are right, it's not worth switching off kondo here just because of that.